### PR TITLE
vm: fix nested tests

### PR DESCRIPTION
**Pull request by gabrocheleau**
This PR "un-nests" some nested `it` vitest tests. Nested `it` functions do not get executed, and therefore "silently pass".

### CODE REVIEW

File: packages/vm/test/api/runBlock.spec.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ import type { TypedTransaction } from '@ethereumjs/tx'
 const testData = require('./testdata/blockchain.json')
 const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Berlin })
 describe('runBlock tests', () => {
-  it('runBlock() -> successful API parameter usage', async () => {
+  describe('runBlock() -> successful API parameter usage', async () => {
```
Comment (acolytec3) [inline packages/vm/test/api/runBlock.spec.ts:30]: What would you say to just removing the nested `describe` anyway?  I don't think it gives us any advantage in this case.

Comment (gabrocheleau) [reply packages/vm/test/api/runBlock.spec.ts:30]: Yeah sure, just wanted to minimally change what was already there, but agree they don't do much, will remove.

File: packages/vm/test/api/runTx.spec.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ const TRANSACTION_TYPES = [
 const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London })
 common.events.setMaxListeners(100)
 describe('runTx tests', () => {
-  it('runTx() -> successful API parameter usage', async () => {
+  describe('runTx() -> successful API parameter usage', async () => {
```
[the remark this answers is not in the document]
Comment (gabrocheleau) [reply packages/vm/test/api/runTx.spec.ts]: Removed the top-level one.

File: packages/vm/test/api/runBlock.spec.ts (hunk 1 above)
Comment (gabrocheleau) [reply packages/vm/test/api/runBlock.spec.ts]: Removed the top-level one.

**Review by jochem-brouwer — APPROVED** [review]
LGTM. I have not compared the diffs but have instead compared the new file to the old file.

**Pull request merged**